<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugStatusScreen</class>
 <widget class="QWidget" name="DebugStatusScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Debug Status Screen</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>settingLabel</zorder>
   <zorder>back</zorder>
  </widget>
  <widget class="QWidget" name="functions" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>320</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QListWidget" name="ipAddresses">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="shutDown">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Shut down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>5</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reboot">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>5</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Reboot</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverURL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true">Server URL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
